enginetest/enginetests.go

```diff
@@ -460,6 +460,22 @@ func TestInsertIntoErrors(t *testing.T, harness Harness) {
 	}
 }
 
+func TestLoadData(t *testing.T, harness Harness) {
+	for _, script := range LoadDataScripts {
+		if !script.Skip {
```

Comment:
See comment below about the right way to do skipped tests.

Also, whenever you skip a test in go, you need to explicitly call t.skip(), not just fail to run it. Otherwise we have no visibility into how many tests are being skipped like this.